src/services/webclient.ts

```diff
@@ -2008,12 +2008,39 @@ export class WebClientService {
             .then((titlePrefix) =>  {
                 const title = `${titlePrefix} ${senderName}`;
                 let body = '';
-                switch (message.type) {
+                let messageType = message.type;
+                let caption = message.caption;
+                let captionString = '';
+                if (typeof caption !== 'undefined') {
```

Comment:
You can use `caption !== undefined` here. The `undefined` object is immutable since ES5, so it should be safe to compare to it directly.